config/initializers/okcomputer.rb

```diff
@@ -0,0 +1,16 @@
+require 'okcomputer'
+
+OkComputer.check_in_parallel = true
+
+##
+# REQUIRED checks
+
+# Simple echo of the VERSION file
+class VersionCheck < OkComputer::AppVersionCheck
```

Comment:
I don't see a `VERSION` file; are there plans to add one, or is the upstream one (https://github.com/sportngin/okcomputer/blob/master/lib/ok_computer/built_in_checks/app_version_check.rb) that checks the capistrano metadata good enough?